codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -89,36 +89,10 @@ def start_worker_job(self):
         )
         # This needs to be a unique directory since Batch jobs may share a host
         work_dir = os.path.join(work_dir_prefix, 'cl_worker_{}_work_dir'.format(worker_id))
-        worker_network_prefix = 'cl_worker_{}_network'.format(worker_id)
-        command = [
-            self.args.worker_executable,
-            '--server',
-            self.args.server,
-            '--verbose',
-            '--exit-when-idle',
-            '--idle-seconds',
-            str(self.args.worker_idle_seconds),
-            '--work-dir',
-            work_dir,
-            '--id',
-            worker_id,
-            '--network-prefix',
-            worker_network_prefix,
-        ]
-        if self.args.worker_tag:
-            command.extend(['--tag', self.args.worker_tag])
-        if self.args.worker_max_work_dir_size:
-            command.extend(['--max-work-dir-size', self.args.worker_max_work_dir_size])
-        if self.args.worker_delete_work_dir_on_exit:
-            command.extend(['--worker-delete-work-dir-on-exit'])
-        if self.args.worker_exit_after_num_runs and self.args.worker_exit_after_num_runs > 0:
-            command.extend(['--exit-after-num-runs', str(self.args.worker_exit_after_num_runs)])
-        if self.args.worker_exit_on_exception:
-            command.extend(['--exit-on-exception'])
+        command = self.build_command(worker_id, work_dir)
+
         if self.args.worker_pass_down_termination:
-            command.extend(['--pass-down-termination'])
-        if self.args.worker_tag_exclusive:
-            command.extend(['--tag-exclusive'])
+            command.append('--pass-down-termination')
```

Comment:
you can probably fold this into `bulid_command` as well? Then, for the slurmworkermanager, additionally append it to the command (i think it's fine if the flag shows up twice).

Reply:
I think that's better. I would have had to repeat the check for `AzureBatchWorkerManager`.